modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -377,3 +388,117 @@ Scanner output rules are used in daemon/socket mode.
 . A file size limit applies both to background jobs and to file uploads.
 . After installing ClamAV and the related tools, you will have two configuration files: `/etc/freshclam.conf` and `/etc/clamd.d/scan.conf`. 
 . We recommend that you enable verbose logging in both `clamd.conf` and `freshclam.conf` until you get any kinks with your ClamAV installation worked out
+
+
+== ICAP
+
+{icap-url}[ICAP] is an open standard supported by many antivirus products. With the release of the _Anti-Virus_ app 1.0.0, other virus scanners beside ClamAV can be used via ICAP if you are running it on an ownCloud Enterprise Edition. Currently the only supported virus scanners are _Kaspersky ScanEngine_ and _McAfee Antivirus_ although far more products might simply work.
```

Comment:
With the release of the _Anti-Virus_ app 1.0.0...
I would avoid to print the version, it has no value.
You also may make `Anti-Virus` a link to the marketplace. Even the final path is not set - can be fixed later.

Reply:
clamav in icap is supported as well - right? at least we know this is working .....

Reply:
@mmattel Well, with older versions there's no ICAP, with future versions more virus scanners might be supported.

@DeepDiver1975 Good point, waiting for information from @pmaier1.

Reply:
- Before version 1.0.0 there is no ICAP at all. So might be worth to point that out, yes.
- I would add "supported **and tested**"

Reply:
Okay, version stays, added 'tested'.